xchange-poloniex2/src/main/java/info/bitrich/xchangestream/poloniex2/PoloniexStreamingMarketDataService.java

```diff
@@ -61,6 +61,10 @@ public PoloniexStreamingMarketDataService(PoloniexStreamingService service, Map<
     public Observable<Ticker> getTicker(CurrencyPair currencyPair, Object... args) {
         final ObjectMapper mapper = StreamingObjectMapperHelper.getObjectMapper();
 
+        Observable<Ticker> tickerObservable = Observable.just(currencyPair)
```

Comment:
This feels like it shouldn't be here. It looks like it will result in a recursive call to itself, and the tickerObservable is never used.

Reply:
Ooh, that definitely shouldn't be there. Crept in while I was checking some example code for one of your PRs....

Reply:
Done, thanks for the spot